app/packs/stylesheets/allCollections.scss

```diff
@@ -24,4 +24,8 @@ $collapse-icon: url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg'
       }
     }
   }
+
+  .dataTable-search {
```

Comment:
The `searching: false` configuration (from the documentation) does not work on DataTables so hiding the search within this page via css was the easiest way to disable it.